release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -73,6 +73,10 @@ With this release, you can now install {op-system} to iSCSI boot devices. Multip
 In {product-title} {product-version}, the installation program uses Cluster API instead of Terraform to provision cluster infrastructure during installations on AWS. There are several additional required permissions as a result of this change. For more information, see xref:../installing/installing_aws/installing-aws-account.adoc#installation-aws-permissions_installing-aws-account[Required AWS permissions for the IAM user].
 
 Additionally, SSH access to control plane and compute machines is no longer open to the machine network, but is restricted to the security groups associated with the control plane and compute plane machines.
+[id="ocp-4-16-installation-and-update-alibaba_{context}"]
+==== Installation on Alibaba Cloud by using Assisted Installer (Technology Preview)
+
+ With this release, {product-title} installation program no longer supports the installer-provisioned installation on Alibaba Cloud platform. You can install a cluster on Alibaba Cloud by using Assisted Installer, which is currently a Technology Preview feature. For more information, see <xref to Alibaba docs in OCP>.
```

Comment:
```suggestion
 With this release, the {product-title} installation program no longer supports installer-provisioned installation on the Alibaba Cloud platform. You can install a cluster on Alibaba Cloud by using Assisted Installer, which is currently a Technology Preview feature. For more information, see <xref to Alibaba docs in OCP>.
```

Reply:
Done. 
For the broken preview, there was a space introduced in the beginning of the sentence which caused the whole block to be rendered as a code. It's strange and took me some time to figure that out. :D